src/grammar.lua

```diff
@@ -453,3 +456,17 @@ function pmatch(patt)
     end
   end)
 end
+
+_M.anyOf	= anyOf
```

Comment:
Done in https://github.com/tst2005/leg/commit/c67b1872a454031ab7f2dc0d7cfda72fc0f6f5d5
Regards,